diff --git a/node_modules/@firebase/auth/dist/node-esm/index-e01c0b19.js b/node_modules/@firebase/auth/dist/node-esm/index-e01c0b19.js
index 063705b..85fcb32 100644
--- a/node_modules/@firebase/auth/dist/node-esm/index-e01c0b19.js
+++ b/node_modules/@firebase/auth/dist/node-esm/index-e01c0b19.js
@@ -3,7 +3,6 @@ import { SDK_VERSION, _getProvider, _registerComponent, registerVersion, getApp
 import { __rest } from 'tslib';
 import { Logger, LogLevel } from '@firebase/logger';
 import { Component } from '@firebase/component';
-import * as fetchImpl from 'node-fetch';
 
 /**
  * @license
@@ -6122,7 +6121,7 @@ function registerAuth(clientPlatform) {
  * limitations under the License.
  */
 // Initialize the fetch polyfill, the types are slightly off so just cast and hope for the best
-FetchProvider.initialize(fetchImpl.default, fetchImpl.Headers, fetchImpl.Response);
+FetchProvider.initialize(fetch, Headers, Response);
 // First, we set up the various platform-specific features for Node (register
 // the version and declare the Node getAuth function)
 function getAuth(app = getApp()) {
diff --git a/node_modules/@firebase/auth/dist/node-esm/index.js b/node_modules/@firebase/auth/dist/node-esm/index.js
index 3acd56d..931e0ad 100644
--- a/node_modules/@firebase/auth/dist/node-esm/index.js
+++ b/node_modules/@firebase/auth/dist/node-esm/index.js
@@ -4,5 +4,5 @@ import '@firebase/app';
 import 'tslib';
 import '@firebase/logger';
 import '@firebase/component';
-import 'node-fetch';
+// import 'node-fetch';
 //# sourceMappingURL=index.js.map
