diff --git a/node_modules/next/navigation-types/compat/navigation.d.ts b/node_modules/next/navigation-types/compat/navigation.d.ts
index 28b46d3..a6fb400 100644
--- a/node_modules/next/navigation-types/compat/navigation.d.ts
+++ b/node_modules/next/navigation-types/compat/navigation.d.ts
@@ -8,7 +8,7 @@ declare module 'next/navigation' {
    * If used from `pages/`, the hook may return `null` when the router is not
    * ready.
    */
-  export function useSearchParams(): ReadonlyURLSearchParams | null
+  export function useSearchParams(): ReadonlyURLSearchParams
 
   /**
    * Get the current pathname. For example, if the URL is
