diff --git a/node_modules/next-auth/core/routes/callback.js b/node_modules/next-auth/core/routes/callback.js
index d30a999..693fa5f 100644
--- a/node_modules/next-auth/core/routes/callback.js
+++ b/node_modules/next-auth/core/routes/callback.js
@@ -390,6 +390,13 @@ async function callback(params) {
       provider: provider.id
     };
 
+    // Added for this specific use case
+    if (provider.id === 'credentials') {
+      account.access_token = user.access_token;
+      account.expires_at = user.expires_at;
+      account.refresh_token = user.refresh_token;
+    }
+
     try {
       const isAllowed = await callbacks.signIn({
         user,
